app/helpers/landing_page_helper.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Landing Page helpers
+module LandingPageHelper
+  def format_hours(hours)
+    return '' unless hours
+
+    return hours.closed_note if hours.closed_note
+
+    return "Open until #{format_time(hours.hours_today.closes_at_time)}" if hours.open_now?
+
+    return "Closed until #{format_time(hours.next_open_hours.opens_at_time)}" if hours.next_open_hours
+
+    'Closed'
+  end
+
+  def format_time(time)
+    hour = time.hour % 12
+    hour = 12 if hour.zero?
+    minute = time.min
+    am_pm = time.hour >= 12 ? 'pm' : 'am'
+
+    time_only = minute.zero? ? "#{hour}#{am_pm}" : "#{hour}:#{format('%02d', minute)}#{am_pm}"
+    return time_only if time.today? || time.tomorrow?
+
+    "#{time_only} #{time.strftime('%A')}"
+  end
+
+  def global_collection_count
```

Comment:
@taylor-steve forgot to comment on this yesterday. Should this move into the controller? I think of helpers as being for formatting information for display and with this we're fetching a value from Solr. Maybe it should also be cached  for an hour or something? It's unlikely to change more than once a day.

Reply:
That makes sense to me. Do you know, do I need to uncomment this line in production.rb or does Rails default to something?

```rb
# config.cache_store = :mem_cache_store
```